output: Retry document-level `429`s by default (#13620)
elastic/apm-server: changelogs/head.asciidoc

==== Intake API Changes

[float]
==== Added

> - APM Server now automatically retries document-level 429s from Elasticsearch to avoid dropping data. This is controlled by `output.elasticsearch.max_retries`, and defaults to `3`. {pull}13620[13620]

Reviewer: nit: make it clear that `output.elasticsearch.max_retries` controls both request-level and document-level retries.
```suggestion
- APM Server now automatically retries document-level 429s from Elasticsearch to avoid dropping data. `output.elasticsearch.max_retries` now controls both request-level and document-level retries, and defaults to `3`. {pull}13620[13620]
```
